plugins/home/src/homePageComponents/VisitedByType/Content.test.tsx

```diff
@@ -117,6 +122,39 @@ describe('<Content kind="recent"/>', () => {
     expect(container.querySelectorAll('li')[0]).toBeVisible();
     expect(container.querySelectorAll('li')[1]).not.toBeVisible();
   });
+
+  it('allows items to be filtered', async () => {
+    const configApiMock = new MockConfigApi({
+      home: {
+        topVisits: {
+          filterBy: [
+            {
+              field: 'pathname',
+              operator: '==',
+              value: '/explore',
+            },
+          ],
+        },
+      },
+    });
+
+    const { getByText, queryByText } = await renderInTestApp(
+      <TestApiProvider
+        apis={[
+          [configApiRef, configApiMock],
+          [visitsApiRef, mockVisitsApi],
+        ]}
+      >
+        <ContextProvider>
+          <Content kind="recent" />
```

Comment:
It would be nice to cover the following four scenarios:

```
when kind === ‘recent’ and home.recentVisits.filterBy is available
when kind === ‘recent’ and home.recentVisits.filterBy is not available
when kind === ‘top’ and home.topVisits.filterBy is available
when kind === ‘top’ and home.topVisits.filterBy is not available
```

Reply:
So I was looking at the test again, we mock the visits API so regardless of the filter I will add in the config the response will be the same as the mock. So I updated the test to check the parameters that `list` is receiving. 
WDYT?

Reply:
@djamaile , any chance you can review my changes?

Reply:
unit tests look good to me.